docs/integration.rst

```diff
@@ -404,6 +405,13 @@ BigQueryDeleteDatasetOperator
 
 .. autoclass:: airflow.contrib.operators.bigquery_operator.BigQueryDeleteDatasetOperator
 
+.. _BigQueryCreateEmptyDatasetOperator:
+
+BigQueryCreateEmptyDatasetOperator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: airflow.contrib.operators.bigquery_operator.BigQueryCreateEmptyDatasetOperator
+
```

Comment:
And can you sort out the order as well?

This:
![image](https://user-images.githubusercontent.com/8811558/45588673-3daa8200-b910-11e8-9754-ea8c37460be6.png)


Should match with:
![image](https://user-images.githubusercontent.com/8811558/45588677-5155e880-b910-11e8-9135-d7a2e9d7e1e9.png)